homeassistant/components/zwave/__init__.py

```diff
@@ -830,6 +830,25 @@ def __init__(self, values, domain):
 
     def network_value_changed(self, value):
         """Handle a value change on the network."""
+        if (value.command_class == const.COMMAND_CLASS_CENTRAL_SCENE and
```

Comment:
Move this logic into the [ZWaveNodeEntity](https://github.com/home-assistant/home-assistant/blob/bd039b8c5332e9225ef2bd8291c9fd12309bebd9/homeassistant/components/zwave/node_entity.py#L76). These events should only fire once per node, not for each entity on the node.

Reply:
I started down that path originally, but in my testing there wasn't a ZWaveNodeEntity instance for the CentralScene value that OZW was sending the ValueChanged notification for.  I'm not familiar enough with the codebase to determine if I should be automatically creating the entity earlier on during the initialization process, but I added the code here as it was the furthest I could trace the ValueChanged from OZW before it was discarded.

Reply:
OK, I think I see part of the problem. The `network_node_changed` is subscribed to a bunch of Z-Wave events.

What you'll want to do is copy [this line](https://github.com/home-assistant/home-assistant/blob/bd039b8c5332e9225ef2bd8291c9fd12309bebd9/homeassistant/components/zwave/__init__.py#L828-L829) into the ZWaveNodeEntity constructor, and add the method as it's defined here, only including your new logic.

That existing method could probably use some cleanup at some point, since it's probably not working as expected.